.github/workflows/push.yaml

```diff
@@ -47,7 +49,11 @@ jobs:
             tmp=${WASM_PATH%%.wasm}
             EXTENSION_NAME=$(basename ${tmp})
             mkdir -p wasm-extensions/${EXTENSION_NAME//_/-}
-            mv bazel-bin/${WASM_PATH} wasm-extensions/${EXTENSION_NAME//_/-}/${SHA}.wasm
+            cp bazel-bin/${WASM_PATH} wasm-extensions/${EXTENSION_NAME//_/-}/${GITHUB_SHA}.wasm
+
+            # also push binary with more readable name, so that it could be easily used in configuration and documentation.
+            cp wasm-extensions/${EXTENSION_NAME//_/-}/${GITHUB_SHA}.wasm wasm-extensions/${EXTENSION_NAME//_/-}/${VERSION}-$(date +"%Y-%m-%d-%H-%M-%S").wasm
```

Comment:
Please add example before and after names.
basic_auth-1.7-$(date) I think

Reply:
Done.